packages/components/src/reaction_icon/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { REACTION_BRAVO, REACTION_GOOD_IDEA, REACTION_LOVE, REACTION_SIZE_M, REACTION_THANK_YOU } from '../constants';
+import styles from './index.module.scss';
+import { ReactComponent as Bravo } from './images/bravo.svg';
+import { ReactComponent as Thankyou } from './images/thankyou.svg';
+import { ReactComponent as GoodIdea } from './images/good_idea.svg';
+import { ReactComponent as Love } from './images/love.svg';
+
+const ICONS = {
+  [REACTION_THANK_YOU]: Thankyou,
+  [REACTION_GOOD_IDEA]: GoodIdea,
+  [REACTION_BRAVO]: Bravo,
+  [REACTION_LOVE]: Love,
+};
+
+const ReactionIcon = ({
+  className: passedClassName,
+  reaction,
+  size = REACTION_SIZE_M,
+  colorize = false,
```

Comment:
the naming is a bit awkward, more grammatically correct would be `colored` I think, which is also awkward. maybe flip it? `simple` to not have colors?

Reply:
I don't think using colors by default is a good choice. I would expect something called `*Icon` to be unicolor by default (like all our icons in @goodhood/icons that are named that way)

`colored` may be better

Reply:
just call it `Reaction` and problem solved